src/SourceBuild/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/ScancodeResults.cs

```diff
@@ -0,0 +1,23 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Text.Json.Serialization;
+
+namespace Microsoft.DotNet.SourceBuild.SmokeTests;
+
+public class ScancodeResults
```

Comment:
Have you consider making these nested types of the test?  If not, IMO you should follow the one type per file style.

Reply:
Fixed in https://github.com/dotnet/installer/pull/17442/commits/b8dafb201f7b9bf0778cd0ac587c146bb49cd4d5